setup.py

```diff
@@ -17,7 +17,7 @@
       author="Daniel Carrión",
       author_email="daniel@carto.com",
       description="SDK around CARTO's APIs",
-      version="1.3.0",
+      version="1.3.1",
```

Comment:
Since this is a new release could you please update the version in conf.py and NEWS.md? See the release process [here](https://github.com/CartoDB/carto-python/blob/master/CONTRIBUTING.md)